src/content/documentation/develop/differences-between-desktop-and-android-extensions.md

```diff
@@ -80,15 +72,14 @@ As a result of the UI differences, extensions for Firefox for Android do not sup
 - [`sidebarAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/sidebarAction) and the related [`sidebar_action`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/sidebar_action) `manifest.json` key, due to the limited screen real estate on Android devices sidebars, such as the browser history, are presented in full browser tabs. Where possible, you should move any sidebar content to tabs as well.
 - [`windows`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/windows) as there is only one Firefox on Android 'window', so it has no ability to open or otherwise manipulate additional browser windows.
 
-Support for [`browserAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction) and the [`browser_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action) `manifest.json` key is under development. Firefox 55 will support `default_title` and `default_popup` of the `manifest.json` key [`browser_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action), using `default_title` to add an item to the Firefox for Android menu, and the [`browserAction.onClicked()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/onClicked) event will be available to listen for the menu item being tapped. Additionally, in Firefox 57 support for the [`browserAction.setTitle`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/setTitle) and [`browserAction.getTitle`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/getTitle) methods will be added.
 
 ### Effect on your add-on UI
 
-These differences impact the way you expose your add-on in the Firefox UI. The most common option, adding a button for your add-on to the Firefox toolbar with [`browserAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction), is not available (at least until Firefox 55). Nor can you expose your add-on through a sidebar or context menu. You will, therefore, use an address bar button (through the `manifest.json` [`page_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action) key and [`pageAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/pageAction) API) remembering that by default this button is hidden and must be shown programmatically.
+You cannot expose your add-on through a sidebar or context menu. You, therefore, use an address bar button (through the `manifest.json` [`action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action) key and [`action`](https://developer.mozilla.org//Add-ons/WebExtensions/API/action) API) remembering that by default this button is hidden and must be shown programmatically.
```

Comment:
action is MV3, continue to refer to MV2 so browser_action here.

Reply:
@mixedpuppy I'm wondering why, as the BCD shows the action is supported on Android? Is there some limitation on MV3 support on Android?

Reply:
@rebloor what @mixedpuppy mentioned is a difference between MV2 and MV3 extensions, which is a difference that both Desktop and Mobile shares.

"action" is the terminology used for MV3 extensions and that is supported by both Desktop and Android , while "browser_action" is the MV2 name for the exact same feature in MV2 (on both Android and Desktop).]

"page_action" stays named as "page_Action" for both MV2 and MV3 in Firefox (while in Chrome MV3 page_action isn't supported at all).

Let me know if this additional detail helped to clarify this point.

Reply:
I have reworked this section reverting the content I incorrectly changed from page action and adding information on the MV2/MV3 browser action/action options.